bin/export_improved_scores

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/python
+# Copyright 2015 Google.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This script exports some improved results (under parameter control).
+#
+import argparse
+import sys
+
+import encoder
+import optimizer
+import pick_codec
+import score_tools
+
+def ImprovementsInOneParameterSet(my_optimizer, filename):
+  my_encoder = encoder.Encoder(my_optimizer.context, filename=filename)
+  cache = my_optimizer.context.cache
+  results = []
+  for encoding in cache.AllScoredEncodingsForEncoder(my_encoder):
+    new_score = my_optimizer.Score(encoding)
+    try:
+      old_score = my_optimizer.Score(encoding, scoredir='score_storage')
+    except (encoder.Error, KeyError):
+      # encoder.Error can happen if old score is broken.
+      # KeyError can happen if old score is missing some now-mandatory
+      # field.
+      # In both cases, the new score should be used.
+      if new_score > 0.0:
+        results.append(encoding)
+      continue
+    if new_score > old_score:
```

Comment:
Suggestion: move this into the try block and remove the continue, should do the same thing right?